app/src/main/java/com/yeonae/chamelezone/view/place/PlaceReviewTabFragment.kt

```diff
@@ -67,7 +67,7 @@ class PlaceReviewTabFragment : Fragment(), PlaceReviewContract.View {
         setAdapter()
 
         presenter = PlaceReviewPresenter(
-            Injection.reviewRepository(), Injection.memberRepository(requireContext()),this
+            Injection.reviewRepository(), Injection.memberRepository(requireContext()), this
```

Comment:
```suggestion
            Injection.reviewRepository(), Injection.memberRepository(APPLICATION_CONTEXT), this
```
fragment의 context를 넣으면 좋지않아요 인자를 아에 제거하고 App context를 사용하는 것이 좋겠어요

Reply:
수정했습니다